src/backends/llk.c

```diff
@@ -530,8 +537,6 @@ static HCountedArray *llk_parse_chunk_(HLLkState *s, const HParser* parser,
   return seq;
 
  no_parse:
```

Comment:
Best to clear the exception handler here; that way we can maintain the invariant that returning to the exception handler is *always* valid.